imports/client/components/CallSection.tsx

```diff
@@ -143,25 +152,28 @@ class RTCCallSection extends React.Component<RTCCallSectionProps> {
           <Button variant="danger" size="sm" onClick={this.leaveCall}>leave call</Button>
         </div>
         <div className="chatter-subsection av-chatters">
-          <header>
+          <header onClick={this.props.onToggleCallersExpanded}>
+            <FontAwesomeIcon fixedWidth icon={callersHeaderIcon} />
             {`${callerCount} caller${callerCount !== 1 ? 's' : ''}`}
           </header>
-          <div className="people-list">
-            {this.renderSelfBox()}
-            {this.props.signalsReady && this.props.selfParticipant && others.map((p) => {
-              return (
-                <CallLinkBox
-                  key={p._id}
-                  rtcConfig={this.props.rtcConfig!}
-                  selfParticipant={this.props.selfParticipant!}
-                  peerParticipant={p}
-                  localStream={this.props.localStream}
-                  audioContext={this.props.audioContext}
-                  deafened={this.props.deafened}
-                />
-              );
-            })}
-          </div>
+          {this.props.callersExpanded && (
```

Comment:
This is not viable as written.  `CallLinkBox` manages the actual audio call lifecycle and audio playback; we can't make the rendering of `CallLinkBox` conditional on collapse state or collapsing can't happen without disconnecting the audio call (but confusingly, still showing you as wanting to participate).

What would you think about making `people-list` have `overflow: auto` and specifying a `max-height` instead?

Reply:
Thanks for catching that. I did not realize CallLinkBox was functional. I'll fix tonight (probably just using `display:none`).